canonical/ubuntu.com
# [WD-11117] add a parameter to YT video URL link to prevent unnecessary suggestions

## Done

Added a `rel=0` parameter to YouTube URL link to prevent videos from other channels to show up in suggestions after the video reaches the end. The solution comes from this [SO thread](https://stackoverflow.com/a/36327162/6379181).

## QA

- Navigate to /core
- Play the YouTube embedded video till the end
- Check all the suggestion videos that pop-up and check if all of them come from Canonical YT channel

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-11117

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
